woocommerce/woocommerce-android
# [HACK Week] Remove Zendesk push notification code

Summary
==========
Removes all unused code related to the old push notifications coming from Zendesk. Now we're using an e-mail approach to the support tickets opened through the app, so the notifications are no longer necessary.

How to Test
==========
1. Simply check if the Support Request form is still working as expected.

Update release notes:
